pkgs/applications/audio/reaper/default.nix

```
@@ -11,7 +11,12 @@
 , xdg-utils
 , which
 
-, jackSupport ? true, libjack2
+, libjack2
+, jackSupport ? true
+
+# There are multiple options for the jack library. Another choice is "pipewire.jack".
+, jackLibrary ? libjack2
```

```suggestion
, jackLibrary
```

And the comment can be moved to all-packages.nix

---

done